File: introducao/01_Resolucao_Competicao.c
#include <stdio.h>
int main() {
    float nota1, nota2;
    int faltas;

    while (1) {
        scanf("%f", &nota1);
        scanf("%f", &nota2);
        scanf("%d", &faltas);

        if (nota1 == 0 && nota2 == 0 && faltas == 0)
            break;

        float media = (nota1 + nota2) / 2;

        if (media >= 6 && faltas <= 10)
            printf("Aprovado\n");
        else
            printf("Reprovado\n");
    }

    return 0;
}

File: introducao/01_Resolucao_Competicao.py
while True:
    nota1, nota2, faltas = map(float, input().split(" ")) # 5.2 6.3 7.0

    if nota1 == 0 and nota2 == 0 and faltas == 0:
        break

    media = (nota1 + nota2) / 2

    if media >= 6 and faltas <= 10:
        print('Aprovado')
    else:
        print('Reprovado')
